ember-toucan-core/src/components/form/radio-group-field.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+import RadioFieldComponent from './radio-field';
+
+import type { ToucanFormRadioFieldComponentSignature } from './radio-field';
+import type { WithBoundArgs } from '@glint/template';
+
+export interface ToucanFormRadioGroupFieldComponentSignature {
+  Element: HTMLFieldSetElement;
+  Args: {
+    error?: string;
```

Comment:
Doesn't have to be in this PR, but just remembered @nicolechung mentioning this in one of her reviews of some headless PR: would be nice to have some JSDoc comments here explaining the args in a short sentence. As this is what people will see when having Glint enabled. And probably they will more likely look only at that, and not consult the actual docs, as we all know how docs get (not) read... 😅

Created: https://github.com/CrowdStrike/ember-toucan-core/issues/73